modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLItemSelectorViewDisplayContext.java

```diff
@@ -351,18 +354,18 @@ public void setShowDragAndDropZone(boolean showDragAndDropZone) {
 		_showDragAndDropZone = showDragAndDropZone;
 	}
 
-	private Optional<Long> _getFileEntryTypeIdOptional() {
+	private Long _getFileEntryTypeIdOptional() {
```

Comment:
@JanisZhang If the method doesn't return an `Optional<>`, the method name should no longer contain "Optional".